meilisearch/tests/snapshot/mod.rs

```diff
@@ -71,7 +80,10 @@ async fn perform_snapshot() {
         // for some reason the db sizes differ. this may be due to the compaction options we have
         // set when performing the snapshot
         //server.stats(),
-        server.tasks(),
+
+        // We can't test all the tasks contained in the snapshot because the on the original instance the snapshotCreation task was added
```

Comment:
Not sure to understand this comment.

Reply:
It says that we can't test **all** the tasks; we have to start at task `4` because the original index contains something like this;
```
5. snapshotCreation
4. documentAddition
3. indexCreation
2. documentAddition
1. indexCreation
```

While the snapshotted index contains something like that;
```
4. documentAddition
3. indexCreation
2. documentAddition
1. indexCreation
```

Reply:
> because the on the original instance the snapshotCreation task was added

I don't understand **this** part specifically 😂